mobile/src/components/interviews/interviews-table/components/label/label.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+
+import { Text } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type Props = {
+  text: string;
+  color?: string;
+};
+
+const Label: FC<Props> = ({ text, color }) => {
+  return (
+    <Text style={{ ...styles.label, ...{ backgroundColor: color } }}>
```

Comment:
How bgColor would be set in case when no color prop is provided? (you'd may have such situation cause in props color is declared as optional)